ui/follow_prompt_mark_ii.qml

```diff
@@ -0,0 +1,90 @@
+/*
+ * Copyright 2018 by Aditya Mehra <aix.m@outlook.com>
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *    http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ *
+ */
+
+import QtQuick 2.4
+import QtQuick.Controls 2.0
+import QtQuick.Layouts 1.4
+
+import Mycroft 1.0 as Mycroft
+
+/* Define a screen instructing user to follow the instructions to connect to wifi */
+Mycroft.Delegate {
+    id: root
+    leftPadding: 0
+    rightPadding: 0
+    topPadding: 0
+    bottomPadding: 0
+    property int gridUnit: Mycroft.Units.gridUnit
+
+    Rectangle {
+        id: followPromptBackground
+        anchors.fill: parent
+        color: "#000000"
+
+        // Image of mobile phone with wifi connection instructions.
+        WifiImage {
+            id: followPromptImage
+            anchors.left: parent.left
+            anchors.leftMargin: gridUnit * 2
+            anchors.top: parent.top
+            anchors.topMargin: gridUnit * 2
+            heightUnits: 28
+            widthUnits: 21
+            imageSource: "images/follow-prompt.png"
+        }
+
+        // Text instructions for this step
+        Item {
+            id: followPromptText
+            anchors.left: followPromptImage.right
+            anchors.leftMargin: gridUnit * 4
+            anchors.top: parent.top
+            anchors.topMargin: gridUnit * 5
+            width: gridUnit * 21
+
+            WifiLabel {
+                id: firstLine
+                text: "Follow the"
```

Comment:
The "_screen.dialog" files were a bit hacky, so if we can get rid of them then great - however this prevents us from properly localizing the Skill into other languages. Is there another way we can handle it?

Reply:
A couple of things here.  One is that a dialog file seems to be a strange place to put text that goes on a screen.  I would like to see the translation functionality not limited to just these types of files.  Second thing is I am concerned that the translated verbiage will not fit well on the screen.  There isn't much room for wording there.  I think we need to take a broader look at how we handle translated language on the display.

Reply:
This seems like a pretty major limitation. I know English is the only language we officially support right now but we have a large global audience and we identified different languages as being a key area we want to support in the future.

Very much agree that we need to look more broadly at how we handle translated content. We're currently only testing with the English variant of things, and for me that's what I mean when I say we only officially support English. We don't actively test, and things might not look perfect in another language, but it might be usable and that's better than nothing.

Reply:
I was just remembering that we talked with @AIIX and @notmart about this type of issue quite a while ago and they pointed us to using Qt translatable strings:
https://doc.qt.io/qt-5/qtquick-internationalization.html

Which I think use `.ts` files:
https://doc.qt.io/qt-5/linguist-ts-file-format.html

I can't find any Mycroft Skills using them at the moment - anyone know of any examples?